spec/plans/discover.fmf

```diff
@@ -76,6 +76,16 @@ description: |
             revision. Defaults to the remote repository's default
             branch if url given or to the current ``HEAD`` if url
             not provided.
+            Additionally, one can set ``ref`` dynamically. This is
+            possible using a special file in tmt format stored in
+            the *default* branch of a tests repository. This special
+            file should contain rules assigning attribute ``ref``
+            in an `adjust` block, for example depending on a test
+            run context.
+            Dynamic ``ref`` assignment is enabled whenever
+            ``.tmt/ref.fmf`` file exists in a test repository or
+            the actual test plan reference has a format
+            ``ref: @FILEPATH``.
```

Comment:
But that way, unless one stops inheritance, this configuration file might become a plan or test. And current experience is that any fmf file is ether plan or test (or story for those who know about them).
Another thing against fmf extension is that you don't read this node from tree but you create separate tree purely from this file.

Reply:
So inheritance should not be a problem unless you would like to define the `ref` in a `main.fmf`. Any file name just works and `ref` would not be inherited it stays within the filename 'namespace'. Hm, do you see any need to add `test`, `story`, `plan` or `execute` keys to the dynamic reference file?

Reply:
Ok, then let's leave it as an option?

Reply:
> Shouldn't we require that the dynamic reference file has an `.fmf` extension? 

Do you mean to rephrase the wording somehow? In specs I have added 
`using a special file in tmt format`
and `.fmf` suffix is apparent from the default location. But in case a user chooses to use different suffix, does that really matter? Would you want to enforce that suffix?